stats_overlay.js

```diff
@@ -0,0 +1,259 @@
+
+
+
+/** @constructor */
+shaka.StatsOverlay = function() {
+
+  /** @private {?shaka.player.Player} */
+  this.player_;
+
+  /** @private {?HTMLElement} */
+  this.canvas_;
+
+  /** @protected {number} */
+  this.width = 200;
+
+  /** @protected {number} */
+  this.height = 100;
+
+  /** @private {?number} */
+  this.startTimestamp;
```

Comment:
Did the linter not complain about private variables that don't follow the naming convention (trailing underscore)?